Resources/Prototypes/Catalog/Fills/Lockers/security.yml

```
@@ -196,6 +197,7 @@
         prob: 0.1
       - id: MedkitOxygenFilled
       - id: MedkitAdvancedFilled
+      - id: BoxBodyBag
```

```suggestion
      - id: BoxBodyBag  # Goobstation
```
*fluoride stares at you